### PR TITLE
Implement DelRule

### PR DESCRIPTION
**Pull request by Dhiver**
I needed to delete a rule via its handle number so I added the DelRule method.

### CODE REVIEW

File: rule.go [hunk 1]
```diff
@@ -105,6 +106,27 @@ func (cc *Conn) AddRule(r *Rule) *Rule {
 	return r
 }
 
+// DelRule delete the specified rule via its handle number
+// https://wiki.nftables.org/wiki-nftables/index.php/Simple_rule_management#Removing_rules
+func (cc *Conn) DelRule(r *Rule, handle uint64) *Rule {
+
```
Comment (stapelberg) [inline rule.go]: nit: remove this empty line please

File: rule.go [hunk 2]
```diff
@@ -105,6 +106,27 @@ func (cc *Conn) AddRule(r *Rule) *Rule {
...
+	data := cc.marshalAttr([]netlink.Attribute{
+		{Type: unix.NFTA_RULE_TABLE, Data: []byte(r.Table.Name + "\x00")},
+		{Type: unix.NFTA_RULE_CHAIN, Data: []byte(r.Chain.Name + "\x00")},
+		{Type: unix.NFTA_RULE_HANDLE, Data: binaryutil.BigEndian.PutUint64(handle)},
```
Comment (sbezverk) [inline rule.go]: How do you retrieve rule's handle? It is possible that I missed it but I did not see api to get it.

**Comment by sbezverk** [comment]
@Dhiver any plans to address comments and add unit test? I also need this functionality urgently.

File: rule.go (hunk 2 above)
Comment (sbezverk) [reply rule.go]: Recent changes expose RuleID, Position and Handle information unblocking DelRule..

**Comment by Dhiver** [comment]
Hey! Thanks for the comments, I'll see what I can do this weekend :)

**Comment by sbezverk** [comment]
@Dhiver if you are short on time, please let me know I will do it, as I have already the code.

**Comment by Dhiver** [comment]
If you already have the code, the sooner the better :) :+1:

**Comment by Dhiver** [comment]
See https://github.com/google/nftables/pull/36 for the implementation with a unit test.

Closing.

**Pull request closed without merging**